### DISCUSSION TITLE
Save results ?

### DISCUSSION THREAD

**Discussion by ChristianBdv** [Alfredredbird/tookie-osint#20] [Q&A] [answered]
Hello Alfred

I'm probably stupid, but I can't find the results file when I do a query!
Please let me know where I can find the result!
Furthermore, congratulations on your project, the results are more exhaustive..
Looking forward to reading you
Christian

**Comment by Alfredredbird** [comment, marked as answer]
Hello Christian, you can find the results file in `./captured/captured.alfred`. I have noticed that the print message doesn't say where it's stored. I apologize for the inconvenience. 
~ Alfred

**Reply by ChristianBdv** [reply]
Thank you Alfred for your quick response, but with a friend we ended up finding where the results were recorded.

Personally, I think it is more appropriate to have the result in a txt file with the name of the target, than to have a compilation of results in the same file. More practical for later operation!

I modified the line accordingly:
196 # creates the save file
**197 file_name = uname**

Furthermore the date in front of the URL, I removed it
227 # opens the save file and writes working sites to it
228 with open(file_path, "a") as f:
229 for site in siteList:
230 siteCount += 1
231 with console.status("Working....") as status:
232 siteN = site["site"]
233 siteNSFW = site["nsfw"]
234 siteErrors = site["errorMessage"]
**235 #f.write(str(date))**
236 Startscan(

The file is cleaner
It's up to you if you modify
Long life to your project
Christian

**Reply by Alfredredbird** [reply]
Hey thanks so much for the recommendation. I have removed the date from the file because it was writing the date top many times. I will be taking your edits into consideration. Thank for your time and effort. :D ~ Alfred
